accounts.md

```diff
@@ -0,0 +1,51 @@
+These are the accounts a dev on the team needs to have to support
+our operations.
+
+# Github
+
+All developers need a github profile handle with a public key to push and pull code to our two Princeton github communities. SSH Access to all servers is also through the use of this public key. 
+
+# Services attached to your Princeton SSO Account (i.e. DUO, Your University Account, the netid account)
+* University Email
+* VPN, aka Secure Remote Access (GlobalProtect Client Preferred)
+* Google Drive
+* Libanswers (Patron facing tickets) - Kevin can grant access
+* SN@P (Staff facing tickets) - Request access from Tracy Hall in Operations
+* Approvals (Library Travel Requests) - should be automatically added to this via
+  Peoplesoft
```

Comment:
Can you push this up a line?  I thought it was something separate, since it was on it's own line